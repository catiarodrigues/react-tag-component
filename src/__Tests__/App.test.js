import React from "react"
import { shallow } from "enzyme"
import App from "../App"

describe("<App /> tests", () => {
  const wrapper = shallow(<App />);
  
  it("renders without errors and match snapshot", () => {
    expect(wrapper).toMatchSnapshot()
  })

  it("renders one TagComponent", () => {
    expect(wrapper.find('TagComponent')).toHaveLength(1);
  })

  it("Renders 1 H1 and 1 H2", () => {
    expect(wrapper.find('h1')).toHaveLength(1)
    expect(wrapper.find('h2')).toHaveLength(1)
  })
})