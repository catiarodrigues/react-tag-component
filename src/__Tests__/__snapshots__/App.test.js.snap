// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> tests renders without errors and match snapshot 1`] = `
<section
  className="section"
>
  <div
    className="container"
  >
    <h1
      className="title"
    >
      Tag React Component
    </h1>
    <h2
      className="subtitle"
    >
      A simple react component where you can add and remove Tags
    </h2>
    <TagComponent />
  </div>
</section>
`;
